src/pages/settings/index.jsx

```diff
@@ -60,6 +82,24 @@ export default function Settings() {
           labelPlacement="end"
         />
       </div>
+      {devMode && (
+        <div>
+          <FormControlLabel
+            checked={showDatabase}
+            onChange={handleDatabaseChange}
+            control={<Switch />}
+            label="Show database"
+            labelPlacement="end"
+          />
+          <FormControlLabel
+            checked={showNutriscore}
+            onChange={handleNutriscoreChange}
+            control={<Switch />}
+            label="Show Nutri-Score game"
+            labelPlacement="end"
+          />
```

Comment:
If you go to object representation, of the visible pages, this part of the code would probably look like that

```suggestion
        {['nutriscore', 'insights'].map(pageUrl => 
        	<FormControlLabel
        		key={pageUrl}
	              checked={visiblePages[pageUrl]}
	              onChange={handleVisiblePagesChange(pageUrl)}
	              control={<Switch />}
	              label={`Show ${pageUrl} page`}
	              labelPlacement="end"
          />)}
```

with `handleVisiblePagesChange = (pageUrl) => (event) => { do the update of pageUrl key depending on the event.target.checked}`